docs/installation/getting-started/maintenance.md

```diff
@@ -12,7 +12,58 @@ modified_at: 2021-07-16
+
+Each code release (there are usually 10-12 of those a year) brings different changes and new features. These are often divided into major and minor changes, along the lines of [semantic versioning](https://semver.org/).
+
+### Code release
+
+Each code release is noted in the [changelog](https://github.com/ckeditor/ckeditor5/blob/stable/CHANGELOG.md) and enumerates all all changes, additions and bug fixes that took place, also highlighting if there are any breaking changes (e.i. changes that make the latest release incompatible with the previous ones code-wise). These are released on [CKEditor 5 npm site](https://www.npmjs.com/package/ckeditor5), as well as {@link installation/advanced/predefined-builds#download-options updated predefined builds} and {@link installation/getting-started/quick-start#creating-custom-builds-with-online-builder Online Builder} and are ready to download.
+
+It is good to follow `nmp` release messages about new packages being published as well as periodicall check the changelog.
+
+### Public release
+
+Soon after the code release a more user-oriented information is published. In a [release blog post](https://ckeditor.com/blog/?category=releases) the latest release version is presented in more details, with examples and links to the new or updated documentation if it brings important information about the release. The release blog post would also provide additional information and context for the changes if needed and direct the user toward migration guides if these were created for the release.
+
+It is good to follow [CKEditor Ecosystem Blog](https://ckeditor.com/blog/) as it also brings other important articles and news about features, changes and planned development.
+
+### Migration guides
+
+Should there be any breaking or important changes that affect the editor code and require special attention, these will also be published in CKEditor 5 documentation in the {@link updating/index Updating section}. These guides will provide a more technical, code-oriented information directed at integrators, administrators and developers and will offer solutions and necessary steps to take while updating.
+
+Administrators and developers should always refer to migration guides after each release and make sure all to implement all the introduced changes properly to ensure stable and uninterrupted operation.
+
+## Daily maintenance
+
+### Upgrade regularly
+
+CKEditor 5 should be upgraded frequently as bug fixes and new features, including support for new browser versions, are not backported. While installing and using CKEditor 5 instance, especially when adding new features, always make sure all the packages are of the same (preferably latest) version. If this requirement is not met, errors may occur.
+
+If you modified any distribution files to add your custom configuration, pay special attention to not overwrite your content (including config.js, contents.css, styles.js and custom template files) when upgrading.
```

Comment:
I think that this sentence doesn't make sense for CKEditor 5, as it has been written from the CKEditor 4 perspective (e.g. the mentioned files don't exist in CKEditor 5).

Reply:
Addressed in the commit.